packages/sidebar/package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "@hixme-ui/sidebar",
+  "version": "1.0.0",
+  "description": "hixme-ui Sidebar component",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "lib/index.js",
+  "scripts": {
+    "clean": "rimraf lib",
+    "test": "jest",
+    "test:watch": "npm test -- --watch --verbose",
```

Comment:
The `--verbose` flag here helps visualize all the tests that are running a bit better.

Reply:
sweet